libs/@hashintel/type-editor/src/entity-type-editor/shared/entity-type-table.tsx

```diff
@@ -37,6 +37,10 @@ import { useIsReadonly } from "../../shared/read-only-context";
  */
 const EDIT_BAR_HEIGHT = 66;
 
+const isSafariBrowser = () =>
+  navigator.userAgent.indexOf("Safari") > -1 &&
+  navigator.userAgent.indexOf("Chrome") <= -1;
```

Comment:
I think this function will crash if called on the server because `navigator` is unavailable – can we add a check for being in the browser?

Reply:
Updated it here https://github.com/hashintel/hash/pull/2594/commits/acc6012598f720f39057bb76c75878ab28996fb2